### ISSUE TITLE
FreeDesktop Flatpak allowlist should be managed by a package

### ISSUE THREAD

**Issue by cassidyjames** [elementary/os#540]
This was great to get up and running, but it seems like it will be problematic to update it in the future if needed.

https://github.com/elementary/os/blob/3530ddd17ef55ff2d9718a1cb95a9908a86a8b0d/etc/config/hooks/live/001-install-flatpaks.chroot#L4-L10

Instead, I imagine this allowlist should be managed by a Debian package—but I'm not sure which package makes the most sense.

**Comment by danrabbit** [comment]
Yeah I was thinking this the other day as well. It should probably go in default settings since we have the local repo stuff there also

**Comment by lenemter** [comment]
Closing because we include Flathub by default now
